clang/include/clang/Lex/Preprocessor.h

```diff
@@ -1914,7 +1909,7 @@ class Preprocessor {
 
   /// Recompute the current lexer kind based on the CurLexer/
   /// CurTokenLexer pointers.
-  void recomputeCurLexerKind();
+  void recomputeCurLexerCallback();
```

Comment:
I think renaming this one is a bit too aggressive. Ideally that we use a callback is an implementation detail that should not leak more than it needs to

 ```suggestion
  void recomputeCurLexerKind();
```